octobot_trading/modes/script_keywords/basic_keywords/amount.py

```diff
@@ -42,6 +42,16 @@ async def get_amount_from_input_amount(
     elif amount_type is dsl.QuantityType.AVAILABLE_PERCENT:
         amount_value = await account_balance.available_account_balance(context, side, reduce_only=reduce_only) \
                        * amount_value / 100
+    elif amount_type is dsl.QuantityType.QUOTE_VALUE:
+        if (context.exchange_manager.is_future 
+            and context.exchange_manager.exchange.is_inverse_symbol(context.symbol)
+            ):
+            # is measured in QUOTE_VALUE
+            pass
+        else:
+            # avoid circular import error
+            import tentacles.Meta.Keywords.scripting_library.data.reading.exchange_public_data as exchange_public_data
```

Comment:
Is there another way to handle imports in the scripting library to avoid circular imports?